index.html

```diff
@@ -0,0 +1,410 @@
+        </div>
+    </div>
+    <div class="container">
+        <div class="advert"><img src="./images/google-ad-1.png" /></div>
```

Comment:
`img` tags are supposed to have `alt` attributes. Please add for all occurrences of `img` tags in your HTML file. 😄